_data/wai-evaluation-tools-list/submissions/e1ede730-0a58-11f0-b2c4-2d841e25aa89.json

```diff
@@ -0,0 +1,64 @@
+{
+  "a11yloc": "https://www.powermapper.com/products/sortsite/vpat/",
+  "actrules": "https://www.w3.org/WAI/standards-guidelines/act/implementations/sortsite/",
+  "assists": [
+    "Generating reports of evaluation results",
+    "Displaying information within web pages",
+    "Creating an accessibility score"
+  ],
+  "automated": [
+    "Single page/screen",
+    "Sample of pages/screens",
+    "Entire website / software / product",
+    "Restricted or password protected pages"
+  ],
+  "comments": "",
+  "contact": "mark.rogers@powermapper.com",
+  "desktop": [
+    "Windows",
+    "MacOS"
+  ],
+  "features": "Desktop product that tests entire websites for WCAG and Section 508 accessibility issues. Also checks for broken links, spelling errors, browser compatibility, HTML validation and CSS validation. The Developer Edition also provides command line / CI integration.\r\n\r\n",
```

Comment:
The list only includes tools for evaluating accessibility.
"Also checks for broken links, spelling errors, browser compatibility, HTML validation and CSS validation." should be removed from the `features` value.